### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/DrizzlePac/align_to_catalogs/align_to_catalogs.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#234]
## What
An accessibility scan flagged 6 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/DrizzlePac/align_to_catalogs/align_to_catalogs.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<table id="table139767837249744" class="table-striped table-bordered table-condensed">` (selector: `#table139767837249744`)
- `<table id="table139767834115664" class="table-striped table-bordered table-condensed">` (selector: `#inspect-the-alignment > .cell.container.docutils > .cell_output.container.docutils > .text_html.output > div > table`)
- `<table id="table139766420122640" class="table-striped table-bordered table-condensed">` (selector: `#table139766420122640`)
- `<table id="table139766427013264" class="table-striped table-bordered table-condensed">` (selector: `#table139766427013264`)
- `<table id="table139766420976976" class="table-striped table-bordered table-condensed">` (selector: `#table139766420976976`)
- `<table id="table139767834115664" class="table-striped table-bordered table-condensed">` (selector: `.cell.container.docutils:nth-child(8) > .cell_output.container.docutils > .text_html.output > div > table`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
